style.css

```diff
@@ -0,0 +1,299 @@
+  --gray6: #5b5b5b;
+  --gray7: #3f3f3f;
+}
+
+*,
+*::after,
+*::before {
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: var(--gray1);
+}
+
+p,
+h2 {
+  background-color: var(--gray1);
+  border-radius: 10px;
+  padding: 15px;
+}
+
+h2 {
+  font-family: "Ubuntu", sans-serif;
+  font-weight: 700;
+}
+
+p {
+  font-family: "PT Serif", serif;
+}
+
+.nav-container {
+  background-color: var(--gray3);
+  height: 100px;
+}
+
+.nav-bar {
+  display: flex;
+  justify-content: space-around;
+  height: 100%;
+  font-family: "Ubuntu", sans-serif;
+  font-weight: 700;
+}
+
+.nav-icon {
+  align-self: center;
+  background-color: var(--gray7);
+  padding: 15px;
+  border-radius: 10px;
+  color: var(--gray1);
+}
+
+.nav-icon:hover {
+  opacity: 90%;
+}
+
+.main-art {
+  display: grid;
+  grid-template-columns: repeat(2, minmax(auto, 1fr));
+  grid-template-rows: repeat(2, minmax(auto, 1fr));
+  height: 800px;
+}
+
+.main-art-items {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  padding: 50px;
+}
+
+.main-art-items:hover {
+  opacity: 90%;
+}
+
+.main-art-items p {
+  margin: 10px;
+}
+
+.main-1 {
+  background-color: var(--gray4);
+}
+
+.main-2 {
+  background-color: var(--gray6);
+}
+
+.main-3 {
+  background-color: var(--gray5);
+}
+
+.main-4 {
+  background-color: var(--gray3);
+}
+
+.main-sub {
+  background-color: var(--gray6);
+  height: 500px;
+  padding: 5% 40% 5% 13%;
+}
+
+.latest-post-grid {
+  display: grid;
+
+  /* grid-gap: 2px; */
+  grid-template-columns: repeat(4, minmax(200px, 1fr));
+  grid-template-rows: repeat(3, minmax(200px, 1fr));
+}
+
+.post-grid-items {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  height: 500px;
+  background-color: var(--gray3);
+  padding: 50px;
+}
+
+.post-grid-items:hover {
+  opacity: 90%;
+}
+
+.post-grid-items p {
+  padding: 15px;
+  margin: 10px;
+}
+
+.four {
+  grid-area: 2 / 1 / span 1 / span 2;
+  background-color: var(--gray6);
+}
+
+.five {
+  grid-area: 2 / 3;
+}
+
+.six {
+  grid-area: 2 / 4;
+}
+
+.seven {
+  grid-area: 3 / 1;
+}
+
+.eight {
+  grid-area: 3 / 2;
+}
+
+.nine {
+  grid-area: 3 / 3 / span 1 / span 2;
+  background-color: var(--gray6);
+}
+
+.add-center {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  height: 500px;
+  background-color: var(--gray4);
+  padding: 50px;
+}
+
+.community-links {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows: repeat(4 1fr);
+  height: 700px;
+}
+
+.cl-one {
+  grid-area: 1 / 1 / span 1 / span 2;
+}
+
+.cl-grid {
+  background-color: var(--gray2);
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+
+  /* padding: 20px; */
+  padding-left: 50px;
+
+  /* margin: 20px; */
```

Comment:
Consider deleting commented out codes. If you need to review a previous version of your code, use git history.

Reply:
Done! thanks @bolah2009